js/apps/admin-ui/src/components/dynamic/url-component.css

```
@@ -0,0 +1,3 @@
+.keycloak__identity-providers__url_component > .pf-v5-c-form__group-control {
```

Do we need a direct child selector, or can we get away with a simple nested one?
```suggestion
.keycloak__identity-providers__url_component .pf-v5-c-form__group-control {
```

---

I followed the same approach selector as in OOTB https://github.com/keycloak/keycloak/blob/main/js/apps/admin-ui/src/identity-providers/add/saml-general-settings.css in order to avoid reinventing the wheel and adding unwanted entropy